boards/samr21-xpro/dist/reset.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# This script resets a CC2538SF53 target using JLink called
```

Comment:
I think at some point there was talk of having a central standard version of these scripts, if so, you should use those and if not you should update this comment to not say CC2538